sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/SessionReceiverManager.cs

```diff
@@ -205,10 +204,11 @@ private async Task CreateReceiver(CancellationToken processorCancellationToken)
 
         private async Task CloseReceiver(CancellationToken cancellationToken)
         {
-            if (_receiver == null || _receiver.IsClosed)
```

Comment:
Out of curiosity... why remove this guard?

Reply:
It should no longer be possible for the receiver to be closed without also being null.